test/JDBC/input/BABEL-CASE_EXPR-vu-verify.sql

```diff
@@ -3106,3 +3106,2464 @@ SELECT CASE 'abcd' COLLATE ARABIC_CI_AS
+   WHEN 1 THEN CAST(N'بطاقة التسجيل - قياسية' AS sys.sysname)
+   WHEN 2 THEN CAST(N'بطاقة التسجيل - قياسية' AS CHAR(100))
+END AS RESULT
+GO
+
+SELECT CASE 2
+   WHEN 1 THEN CAST(N'بطاقة التسجيل - قياسية' AS sys.sysname)
+   WHEN 2 THEN CAST(N'بطاقة التسجيل - قياسية' AS NCHAR(100))
+END AS RESULT
+GO
+
+SELECT CASE 2
+   WHEN 1 THEN CAST(N'بطاقة التسجيل - قياسية' AS sys.sysname)
+   WHEN 2 THEN CAST(N'بطاقة التسجيل - قياسية' AS TEXT)
+END AS RESULT
+GO
+
+SELECT CASE 2
+   WHEN 1 THEN CAST(N'بطاقة التسجيل - قياسية' AS sys.sysname)
+   WHEN 2 THEN CAST(N'بطاقة التسجيل - قياسية' AS NTEXT)
+END AS RESULT
+GO
+
+SELECT CASE 2
+   WHEN 1 THEN CAST(N'登録カード–標準' AS sys.sysname)
+   WHEN 2 THEN CAST(N'登録カード–標準' AS VARCHAR(100))
+END AS RESULT
+GO
+
+SELECT CASE 2
+   WHEN 1 THEN CAST(N'登録カード–標準' AS sys.sysname)
+   WHEN 2 THEN CAST(N'登録カード–標準' AS CHAR(100))
+END AS RESULT
+GO
+
+SELECT CASE 2
+   WHEN 1 THEN CAST(N'登録カード–標準' AS sys.sysname)
+   WHEN 2 THEN CAST(N'登録カード–標準' AS NCHAR(100))
+END AS RESULT
+GO
+
+SELECT CASE 2
+   WHEN 1 THEN CAST(N'登録カード–標準' AS sys.sysname)
+   WHEN 2 THEN CAST(N'登録カード–標準' AS TEXT)
+END AS RESULT
+GO
+
+SELECT CASE 2
+   WHEN 1 THEN CAST(N'登録カード–標準' AS sys.sysname)
+   WHEN 2 THEN CAST(N'登録カード–標準' AS NTEXT)
+END AS RESULT
+GO
+
+SELECT CASE 1
+   WHEN 1 THEN 'abc'
+   WHEN 2 THEN CAST('char' AS sys.sysname)
+END AS RESULT
+GO
+
+SELECT CASE 1
+   WHEN 1 THEN 'abc'
+   WHEN 2 THEN CAST('char0' AS sys.sysname)
+   WHEN 3 THEN CAST('char1' AS VARCHAR(100))
+END AS RESULT
+GO
+
+SELECT CASE 1
+   WHEN 1 THEN 'abc'
+   WHEN 2 THEN CAST('char0' AS sys.sysname)
+   WHEN 3 THEN CAST('char1' AS CHAR(100))
+END AS RESULT
+GO
+
+SELECT CASE 1
+   WHEN 1 THEN 'abc'
+   WHEN 2 THEN CAST('char0' AS sys.sysname)
+   WHEN 3 THEN CAST('char1' AS NCHAR(100))
+END AS RESULT
+GO
+
+SELECT CASE 1
+   WHEN 1 THEN 'abc'
+   WHEN 2 THEN CAST('char0' AS sys.sysname)
+   WHEN 3 THEN CAST('char1' AS TEXT)
+END AS RESULT
+GO
+
+SELECT CASE 1
+   WHEN 1 THEN 'abc'
+   WHEN 2 THEN CAST('char0' AS sys.sysname)
+   WHEN 3 THEN CAST('char1' AS NTEXT)
+END AS RESULT
+GO
+
+SELECT CASE 2
+   WHEN 1 THEN 'abc'
+   WHEN 2 THEN CAST('char' AS sys.sysname)
+END AS RESULT
+GO
+
+SELECT CASE 2
+   WHEN 1 THEN 'abc'
+   WHEN 2 THEN CAST('char0' AS sys.sysname)
+   WHEN 3 THEN CAST('char1' AS VARCHAR(100))
+END AS RESULT
+GO
+
+SELECT CASE 2
+   WHEN 1 THEN 'abc'
+   WHEN 2 THEN CAST('char0' AS sys.sysname)
+   WHEN 3 THEN CAST('char1' AS CHAR(100))
+END AS RESULT
+GO
+
+SELECT CASE 2
+   WHEN 1 THEN 'abc'
+   WHEN 2 THEN CAST('char0' AS sys.sysname)
+   WHEN 3 THEN CAST('char1' AS NCHAR(100))
+END AS RESULT
+GO
+
+SELECT CASE 2
+   WHEN 1 THEN 'abc'
+   WHEN 2 THEN CAST('char0' AS sys.sysname)
+   WHEN 3 THEN CAST('char1' AS TEXT)
+END AS RESULT
+GO
+
+SELECT CASE 2
+   WHEN 1 THEN 'abc'
+   WHEN 2 THEN CAST('char0' AS sys.sysname)
+   WHEN 3 THEN CAST('char1' AS NTEXT)
+END AS RESULT
+GO
+
+--- UDT testing
+create type BABEL_5103_UDT from NVARCHAR(100);
```

Comment:
Lets use meaningful name for UDT.

Reply:
Chnaged name to BABEL_CASE_EXPR_TEST_UDT.